scripts/run_lighthouse_tests.py

```diff
@@ -117,17 +164,17 @@ def enable_webpages():
 
     pattern = 'CONTRIBUTOR_DASHBOARD_ENABLED = .*'
     replace = 'CONTRIBUTOR_DASHBOARD_ENABLED = True'
-    common.inplace_replace_file(FECONF_FILE_PATH, pattern, replace)
+    common.inplace_replace_file(common.FECONF_PATH, pattern, replace)
 
     pattern = '"ENABLE_ACCOUNT_DELETION": .*'
     replace = '"ENABLE_ACCOUNT_DELETION": true,'
-    common.inplace_replace_file(CONSTANTS_FILE_PATH, pattern, replace)
+    common.inplace_replace_file(common.CONSTANTS_FILE_PATH, pattern, replace)
 
 
-def start_google_app_engine_server():
+def start_google_app_engine_server(is_performance_mode):
```

Comment:
Missing Args in docstring.

Also, similarly -- have two constants at top of file: SERVER_MODE_DEV and SERVER_MODE_PROD, and pass in one of these as server_mode. Have a dict that maps the server mode to the app yaml filepath and use that so that you don't have a "default else".

Reply:
Done